keps/sig-instrumentation/1602-structured-logging/kep.yaml

```diff
@@ -0,0 +1,27 @@
+title: Structured Logging
+kep-number: 1602
+authors:
+  - "@jane.doe"
```

Comment:
should this have the same authors as the original?

Reply:
Fixed